app/assets/javascripts/organisations.js.coffee

```diff
@@ -2,4 +2,4 @@
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://jashkenas.github.com/coffee-script/
 jQuery ->
-  $("input[type='checkbox'].publish").bootstrapSwitch(size: 'mini', onText: 'public', offText: 'private')
+  $("input[type='checkbox'].publish").bootstrapSwitch(size: 'mini', onText: 'public',offText: 'private')
```

Comment:
@decareano you've accidentally deleted a single space - please revert and watch out for that - you only want substantive changes in the PR, and this is also breaking the consistency of the code layout ...